esmvaltool/recipes/recipe_climwip_brunner20esd.yml

```diff
@@ -0,0 +1,202 @@
+# ESMValTool
+# recipe_climwip.yml
+---
+documentation:
+  description: EUCP ClimWIP
+
+  authors:
+    - kalverla_peter
+    - smeets_stef
+    - brunner_lukas
+    - camphuijsen_jaro
+    - lorenz_ruth
+
+  maintainer:
+    - kalverla_peter
+    - smeets_stef
+    - brunner_lukas
+    - lorenz_ruth
+
+  references:
+    - brunner2019
+    - lorenz2018
+    - knutti2017
+
+datasets: &model_data
+  - {project: CMIP6, exp: [historical, ssp585], dataset: ACCESS-CM2, ensemble: r(1:3)i1p1f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: ACCESS-ESM1-5, ensemble: r(1:10)i1p1f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: AWI-CM-1-1-MR, ensemble: r1i1p1f1, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: BCC-CSM2-MR, ensemble: r1i1p1f1, grid: gn}
+# ... add more models
+  - {project: CMIP6, exp: [historical, ssp585], dataset: UKESM1-0-LL, ensemble: r(1:4)i1p1f2, grid: gn}
+  - {project: CMIP6, exp: [historical, ssp585], dataset: UKESM1-0-LL, ensemble: r8i1p1f2, grid: gn}
+
+obs_data: &obs_data  # for climwip performance metrics
+  - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3}
+
+preprocessors:
+  climwip_general: &general
+    regrid:
+      target_grid: 2.5x2.5
+      scheme: linear
+
+  climatological_mean:
+    <<: *general
+    climate_statistics:
+      operator: mean
+
+  detrended_std:
+    custom_order: true
+    <<: *general
+    annual_statistics:
+      operator: mean
+    detrend:
+      dimension: time
+      method: linear
+    climate_statistics:
+      operator: std_dev
+
+  annual_trend:
+    <<: *general
+    annual_statistics:
+      operator: mean
+    linear_trend:
+      coordinate: time
+    convert_units:
+      units: "K year-1"
+
+  global_mean:
+    <<: *general
+    climate_statistics:
+      operator: mean
+    area_statistics:
+      operator: mean
+
+  temperature_anomalies:
+    custom_order: true
+    area_statistics:
+      operator: mean
+    annual_statistics:
+      operator: mean
+    anomalies:
+      period: full
+      reference:
+        start_year: 1995
+        start_month: 1
+        start_day: 1
+        end_year: 2014
+        end_month: 12
+        end_day: 31
+      standardize: false
+
+diagnostics:
+  # if local anomalies are needed as predictors they need to be calculated first
+  calculate_local_anomalies_tas:
+    variables:
+      tas_CLIM: &performance_settings
+        short_name: tas
+        start_year: 1995
+        end_year: 2014
+        mip: Amon
+        preprocessor: climatological_mean
+        additional_datasets: *obs_data
+      tas_GLOBAL:
+        <<: *performance_settings
+        preprocessor: global_mean
+    scripts:
+      local_anomalies:
+        script: weighting/calculate_difference_variable_group.py
+        obs_data: native6
```

Comment:
It would be nice if we could do this for all (relevant) variable groups at once.